examples/stable_diffusion_v2/ldm/models/diffusion/dpm_solver/dpm_solver.py

```diff
@@ -112,7 +112,7 @@ def __init__(
             self.total_N = len(log_alphas)
             self.T = 1.0
             self.t_array = self.cast(
-                ops.linspace(ms.Tensor(0.0, ms.float32), ms.Tensor(1.0, ms.float32), self.total_N + 1)[1:].reshape(
+                ms.numpy.linspace(ms.Tensor(0.0, ms.float32), ms.Tensor(1.0, ms.float32), self.total_N + 1)[1:].reshape(
```

Comment:
When you create a new PR, please describe what you have changed and why, so that others don't have to guess. What is the problem with `ops` in this context?